dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -737,15 +737,30 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
     @RolesAllowed("admin")
     @Path("/updateUserMetadataToGetIds")
     @Deprecated
-    @Operation(operationId = "updateUserMetadataToGetIds", description = "Update metadata of all GitHub users and returns list of Dockstore users who could not be updated.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
-    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "Get list Dockstore users we were unable to get GitHub IDs for.", content = @Content(
+    @Operation(operationId = "updateUserMetadataToGetIds", description = "Attempt to update the metadata of all GitHub and Google users and then returns a list of Dockstore users who could not be updated.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "Get list Dockstore users we were unable to get GitHub/Google IDs for.", content = @Content(
             mediaType = "application/json",
             array = @ArraySchema(schema = @Schema(implementation = User.class))))
     @ApiOperation(value = "See OpenApi for details", hidden = true)
     public List<User> updateUserMetadataToGetIds(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user) {
+        List<Token> googleTokens = tokenDAO.findAllGoogleTokens();
         List<Token> gitHubTokens = tokenDAO.findAllGitHubTokens();
         List<User> usersNotUpdatedWithToken = new ArrayList<>();
-        List<User> usersNotUpdatedWithTokenOrUsername = new ArrayList<>();
+        List<User> usersCouldNotBeUpdated = new ArrayList<>();
```

Comment:
I think this is only used for Google users. I suggest renaming to something like `googleUsersCouldNotBeUpdated`, and maybe adding `gitHub` to the previous two, if applicable. It's a little hard to follow which is which when reading.